packages/apps/graph/src/graph/objects/transfer.ts

```diff
@@ -25,6 +22,25 @@ export default builder.prismaNode(Prisma.ModelName.Transfer, {
     moduleName: t.exposeString('moduleName'),
     requestKey: t.exposeString('requestKey'),
     receiverAccount: t.exposeString('receiverAccount'),
+    height: t.expose('height', { type: 'BigInt' }),
+    blockHash: t.exposeString('blockHash'),
+    chainId: t.expose('chainId', { type: 'BigInt' }),
+
+    // blockHash: t.string({
+    //   deprecationReason: 'Use `block.hash` field instead.',
```

Comment:
do we still need this?

Reply:
Good one. But not for the reason you think. I need to add the deprecation message to the normal fields. Then we'll upgrade to the ones that are commented. Then we'll remove them